optuna_dashboard/ts/components/GraphHistory.tsx

```diff
@@ -17,18 +17,25 @@ import {
 } from "@mui/material"
 import { plotlyDarkTemplate } from "./PlotlyDarkMode"
 import {
-  useFilteredTrials,
+  useFilteredTrialsFromStudies,
   Target,
   useObjectiveAndUserAttrTargets,
 } from "../trialFilter"
 
 const plotDomId = "graph-history"
 
+interface HistoryPlotInfo {
+  study_name: string
+  trials: Trial[]
+  directions: StudyDirection[]
+  objective_names?: string[]
```

Comment:
It looks that only `historyPlotInfos[0].objective_names` is used, do you think we need to add `objective_names` here?

Reply:
In this part, objective_names of all studies are used.

https://github.com/optuna/optuna-dashboard/blob/a9517b4cdc5d371d5bcf6c2bf0b5dcc2818c9a33/optuna_dashboard/ts/components/GraphHistory.tsx#L265

Reply:
I see. That makes sense!

Reply:
What do you think about directions?

Reply:
`directions` are also used to make a best line for each study now.

Reply:
I see. Let me resolve this conversation 👍